api/src/org/apache/cloudstack/api/BaseCmd.java

```diff
@@ -192,6 +191,8 @@
     public AlertService _alertSvc;
     @Inject
     public UUIDManager _uuidMgr;
+    @Inject
+    public AnnotationService annotationService;
```

Comment:
As discussed, this is a breakage of pattern in the code base that goes two ways. Adhering to one and not the other has downsides either way. Not important now (in this PR) but worth revisiting is how to poetically describe used services per API without having to describe the samething over and over per set of API, i.e. the Annotation APIs.